docs/architecture/grpc-for-wcf-developers/application-performance-management.md

```diff
@@ -99,48 +95,48 @@ public class StockData : Stocks.StocksBase
 
-Because DiagnosticSource is a part of the core framework, it's supported by several core components, including <xref:System.Net.Http.HttpClient>, Entity Framework Core and ASP.NET Core, including explicit support in the gRPC framework. When ASP.NET Core receives a request, it  checks for a pair of HTTP headers matching the [W3C Trace Context](https://www.w3.org/TR/trace-context) standard. If the headers are found, an Activity is started using the identity values and context from the headers. If no headers are found, an Activity is started with generated identity values that match the standard format. Any diagnostics generated by the framework or by application code during the lifetime of this Activity can be tagged with the trace and span identifiers. The `HttpClient` support extends this further by checking for a current Activity on every request and automatically adding the trace headers to the outgoing request.
+Because `DiagnosticSource` is a part of the core framework, it's supported by several core components. These include <xref:System.Net.Http.HttpClient>, Entity Framework Core, and ASP.NET Core, including explicit support in the gRPC framework. When ASP.NET Core receives a request, it checks for a pair of HTTP headers matching the [W3C Trace Context](https://www.w3.org/TR/trace-context) standard. If the headers are found, an activity is started by using the identity values and context from the headers. If no headers are found, an activity is started with generated identity values that match the standard format. Any diagnostics generated by the framework or by application code during the lifetime of this activity can be tagged with the trace and span identifiers. The `HttpClient` support extends this further by checking for a current activity on every request, and automatically adding the trace headers to the outgoing request.
 
-The ASP.NET Core gRPC client and server libraries include explicit support for DiagnosticSource and Activity, and will create activities and apply and use header information automatically.
+The ASP.NET Core gRPC client and server libraries include explicit support for `DiagnosticSource` and `Activity`, and create activities and apply and use header information automatically.
 
 > [!NOTE]
-> All of this only happens if a *listener* is consuming the diagnostic information. If there's no listener, no diagnostics are written and no activities are created.
+> All of this happens only if a listener is consuming the diagnostic information. If there's no listener, no diagnostics are written and no activities are created.
 
-### Add your own DiagnosticSources and Activities
+### Add your own diagnostic sources and activities
```

Comment:
`DiagnosticSource` and `System.Diagnostics.Activity` are types. I don't think they should be lower case

Reply:
It's weird that they would be written in plural form before if they are API names. 

So maybe

```suggestion
### Add your own `DiagnosticSource` and `Activity`
```